docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install ott-jax
</pre><p>I'd put this on a separate cell above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>